### PR TITLE
Work and about me sections

### PR DESCRIPTION
**Pull request by EvansSnave**
Added the work sections and about me section with functional buttons

### CODE REVIEW

File: Index.html [hunk 1]
```diff
@@ -27,5 +27,113 @@ <h4 class="connect">LET'S CONNECT</h4>
       <li><img src="Assests/Vectorangerllist.svg"></li>
     </ul>
   </section>
+  <section class="worksSection">
+    <div class="space"></div>
```
Comment (Shedrack-Sunday) [inline Index.html:31]: - [ ] Nice effort. but there seems to be a missing margin-top from your header to the work section. This does not give a good view or a better user experience. Kindly stick to the design to make it look great 👍 

## Your view:
image

## Figma:
image

File: Index.html [hunk 2]
```diff
@@ -27,5 +27,113 @@ <h4 class="connect">LET'S CONNECT</h4>
...
+          <ul class="subLanguages">
+            <li class="iconw"><img class="image" src="Assests/Ellipse 1jsLogo.svg">JavaScript</li>
+            <li class="iconw"><img class="image" src="Assests/Ellipse 1htmlLogo.svg">HTML</li>
+            <li class="iconw"><img class="image" src="Assests/Ellipse 1css.svg">CSS</li>
+          </ul>
+        </li>
+        <li class="frameworks languages">Frameworks<img class="droplet side" src="Assests/droplet.png"></li>
+        <li class="skills languages">Skills<img class="droplet sideTwo" src="Assests/droplet.png"></li>
+      </ul>
+    </div>
```
Comment (Shedrack-Sunday) [inline Index.html:136]: - [ ] Nice efforts here 👏 . However, you have a better design that will give a nice user experience and also stick to the to design, kindly take note of the paddings, and margins provided on the original design 👍 
- For the button, you can see the height and width of yours is a bit adjusted. do make sure it has same dimensions as te design given 👍 

## Your design:
image

## Figma:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @EvansSnave      👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 


## Highlights 
- Linters fully set✔️  
- Moderate design✔️  
  
# Required Changes  🌵 

- [x] You missed a couple of design specifications that are of great importance. do note of my suggestions, and correct the accordingly 
- [x] In your README, kindly delete the text below the license file that shows instructions. it makes your code look clustered 👍 
- Your license file is breaking because you have `LICENSE` in the path, but you have `MIT.md on the root

## LICENSE view 
image

_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by EvansSnave** [comment]
I finished the changes but I cannot request another review in my dashboard

File: Index.html [hunk 3]
```diff
@@ -27,5 +27,113 @@ <h4 class="connect">LET'S CONNECT</h4>
...
+        <li class="counter"><img src="Assests/Countercounter.svg"></img></li>
+        <li class="year">2015</li>
+      </ul>
+      <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <div class="works">
+        <button class="htmlButton buttons"></button>
+        <button class="cssButton buttons"></button>
+        <button class="jsButton buttons"></button>
+      </div>
+      <button class="seeProject">See project</button>
```
Comment (Efiamotu-1) [inline Index.html:88]: [OPTIONAL]
- [ ] please make sure your button has the exact shape and size like the figma template below is how the button should look and how your button currently looks. the correct specifications for this button can be found [here](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&t=Mr0WKx6bVRIlwjv8-0) kindly look at them and add them to your css.
- figma design
f-b-1
- your design
s-b-1

File: Index.html [hunk 4]
```diff
@@ -27,5 +27,113 @@ <h4 class="connect">LET'S CONNECT</h4>
       <li><img src="Assests/Vectorangerllist.svg"></li>
     </ul>
   </section>
+  <section class="worksSection">
+    <div class="space"></div>
+    <div class="firstCard">
```
Comment (Efiamotu-1) [inline Index.html:32]: [OPTIONAL]
- [ ] your project image doesnt span across the card, please see the figma template and work on this change 
- figma design
i-n-f
- your design
i-n-s

File: Index.html [hunk 5]
```diff
@@ -27,5 +27,113 @@ <h4 class="connect">LET'S CONNECT</h4>
...
+      <div class="snapshotThree"></div>
+      <h3 class="tonic">Tonic</h3>
+      <ul class="canopyList">
+        <li class="canopy">CANOPY</li>
+        <li class="counter"><img src="Assests/Countercounter.svg"></img></li>
+        <li class="backEnd">Back End Dev</li>
+        <li class="counter"><img src="Assests/Countercounter.svg"></img></li>
+        <li class="year">2015</li>
+      </ul>
+      <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
```
Comment (Efiamotu-1) [inline Index.html:82]: [OPTIONAL]
- [ ] please make sure to use the correct font color to style this paragraph it is important to follow the design patterns provided to you on figma :+1: 
- figma design
f-c-ff
- your design
f-c-f

**Review by Efiamotu-1 — APPROVED** [review]
Hi @EvansSnave , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# STATUS: APPROVED 🕺🥳 

giphy


## Highlights :dart: 
* Correct GitHub flow :heavy_check_mark: 
* Good commit messages :heavy_check_mark: 
* All linters passing :heavy_check_mark: 


- [ ] you should please make your pr summary very descriptive in your next projects :+1: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, you can now Merge since it is Approved ✅, However if there are any conflicts please resolve them before merging.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by EvansSnave** [comment]
Thank you so much!

**Pull request merged**